reference/src/representation/structs-and-tuples.md

```diff
@@ -130,8 +130,9 @@ unsizing coercion may be applied.
 ### C-compatible layout ("repr C")
 
 For structs tagged `#[repr(C)]`, the compiler will apply a C-like
-layout scheme (see section 6.7.2.1 of the [C17 specification][C17] for
-a detailed write-up):
+layout scheme. See section 6.7.2.1 of the [C17 specification][C17] for
+a detailed write-up of what such rules entail. For most platforms,
+however, this means the following:
```

Comment:
Maybe we should just go one step further here and just state that the "most recent" C specification applies. It would be a pain to have to update this every N years, history has shown that the newer C specs are backwards compatible with the old ones (e.g. the C99, C11, and C17 specs have never introduced breaking changes here - they just have defined behavior that was undefined before), and we probably want to be as compatible as possible with C anyways. 

This kinds of tides Rust with the C spec, but that just means that the Rust language would be just another stakeholder in the C standardization process, just like C++ is. They don't really meat that frequently, but we should probably be sending someone there anyways to make sure that the interest of Rust are represented.